src/doc/reference.md

```diff
@@ -1047,11 +1047,8 @@ This is a list of behavior not considered *unsafe* in Rust terms, but that may
 be undesired.
 
 * Deadlocks
-* Reading data from private fields (`std::repr`)
```

Comment:
What's the difference between a private field `bool = 3`, and the same field being `mem::uninitialized`? Presumably it's essentially illegal to read such a thing in both cases, but they should be fine to be "write only".